rrfs-test/validated_yamls/templates/obtype_config/ztd_ukmo.yaml

```diff
@@ -0,0 +1,70 @@
+     - obs space:
+         name: Groundgnss  
+         distribution:
+           name: "@DISTRIBUTION@"
+           halo size: 100e3
+         obsdatain:
+           engine:
+             type: H5File
+             obsfile: single_ztd_obs_2024052700.nc
+#            obsfile: ioda/2024052700.rap.t00z.ztd.tm00.bufr_d.nc
+         obsdataout:
+           engine:
+             type: H5File
+             obsfile: ztd_single_diag_out_2024052700.nc4
+#-------------------------------------------------------
+#         io pool:
+## to be consistent with the ntask number
+#           max pool size: 120
+#           write multiple files: true
+#-------------------------------------------------------
```

Comment:
Do we need this here? Seems like that part might have been removed.

Reply:
These outputs are for debugging purpose. Commented for now.